src/pages/product-details/product-submit/product-submit.js

```diff
@@ -8,30 +8,37 @@ import FavouriteBorderIcon from '@material-ui/icons/FavoriteBorder';
 import Button from '@material-ui/core/Button';
 import { useStyles } from './product-submit.styles';
 
+import { selectLanguageProductsUserWishlist } from '../../../redux/selectors/multiple.selectors';
+
 import {
   addItemToWishlist,
   removeItemFromWishlist
 } from '../../../redux/wishlist/wishlist.actions';
 import { addItemToCart } from '../../../redux/cart/cart.actions';
+import {
+  setToastMessage,
+  setToastSettings
+} from '../../../redux/toast/toast.actions';
 
 import {
   PDP_BUTTONS,
   TOOLTIPS
 } from '../../../translations/product-details.translations';
 
+import { TOAST_MESSAGE } from '../../../translations/toast.translations';
+
 const ProductSubmit = ({ setSizeIsNotSelectedError, sizes }) => {
   const styles = useStyles();
   const dispatch = useDispatch();
   const { language, productToSend, product, wishlistItems } = useSelector(
-    ({ Language, Products, User, Wishlist }) => ({
-      language: Language.language,
-      productToSend: Products.productToSend,
-      product: Products.product,
-      userData: User.userData,
-      wishlistItems: Wishlist.list
-    })
+    selectLanguageProductsUserWishlist
   );
 
+  const toastSettings = {
```

Comment:
I think it's better to move this to separate file.

Reply:
done